certs/manager_test.go

```diff
@@ -121,17 +122,30 @@ func TestAddCertificate(t *testing.T) {
 	cert2Raw, _ := pem.Decode(cert2Pem)
 	cert2ID := HexSHA256(cert2Raw.Bytes)
 
+	buf := new(bytes.Buffer)
+
+	f, err := os.Open("testdata/expired-cert.pem")
```

Comment:
Needs be replaced by dynamically generated certificate. See `genCertificate` usage in cert package.

Reply:
Done in https://github.com/TykTechnologies/tyk/pull/2712/commits/d84230485dae20592cd11acf67473d7bbe89cbee